packages/@orchidui-vue/src/DataDisplay/BeneficiaryCard/OcBeneficiaryCard.vue

```diff
@@ -0,0 +1,61 @@
+<script setup>
+import { Avatar, ListDetail, Icon } from "@/orchidui";
+
+defineProps({
+  variant: {
+    type: String,
+    default: "small",
+    validator: (val) => ["small", "big"].includes(val),
+  },
+
+  beneficiary: {
+    type: Object,
+    required: true,
+  },
+
+  closable: Boolean,
+});
+
+const emit = defineEmits(["removeBeneficiary"]);
+</script>
+
+<template>
+  <div
+    class="rounded shrink-0 border border-oc-gray-200 bg-oc-bg-light px-4 py-5 relative"
+  >
+    <Icon
+      v-if="closable"
+      class="absolute -right-1.5 -top-1.5 border-1 bg-white border-white overflow-hidden rounded-full text-gray-500 cursor-pointer transition-all duration-300 hover:text-oc-error"
+      name="filled-x-circle"
+      @click="emit('removeBeneficiary')"
+    />
+
+    <div class="flex flex-col items-center justify-center gap-y-3">
+      <!--  Main  -->
+      <div class="w-full flex gap-x-3 items-center">
```

Comment:
this component like customer card , https://orchid-git-feature-hit-5707-send-money-components-hit-pay.vercel.app/storybook/?path=%2Fdocs%2Fdatadisplay-customercard-ocoverview--docs

Reply:
Yeah it is similar but they have a lot of small differences. I thought it would make sense to have it as a separate component because the `CustomerCard` would become quite complex with a lot of conditional rendering.

Reply:
@bdebashi to create a variant you can add props: variant="variantName". and we no need new similar  component. cc @spacemonkey92 @mykhailo-diakovych

Reply:
@kewcoder. Updated to use `CustomerCard` for beneficiary